lib/rules/length-zero-no-unit/index.js

```diff
@@ -65,6 +65,11 @@ const rule = function(actual, secondary, context) {
 		});
 
 		root.walkAtRules((atRule) => {
+			// Ignore Less variables
+			if (atRule.variable && !hasBlock(atRule)) {
+				return;
+			}
```

Comment:
Maybe we should move this to utils?

Reply:
Sounds reasonable.
Added util function and some tests.